docs/src/design-system/building-blocks/icons/color.tsx

```diff
@@ -94,23 +94,25 @@ const IconsGroup: React.FC<{
               variant="body2"
               sx={{
                 ...LABEL_STYLE,
-                color: background === 'dark' ? 'inherit' : palette.grey[900],
+                color: background === 'dark' ? palette.grey[50] : palette.grey[900],
               }}
             >
               {COLOR.labels[3]}
             </Typography>
           </Box>
           <Box sx={ICON_WITH_LABEL_STYLE}>
             <Box
-              sx={{ color: background === 'dark' ? palette.grey[50] : palette.grey[300] }}
+              sx={{
+                color: background === 'dark' ? palette.grey[50] : palette.grey[300],
+              }}
             >
               <Icon.Cog />
             </Box>
             <Typography
               variant="body2"
               sx={{
                 ...LABEL_STYLE,
-                color: background === 'dark' ? 'inherit' : palette.grey[900],
+                color: background === 'dark' ? palette.grey[50] : palette.grey[900],
```

Comment:
Since this logic repeats itself, can it be moved to a function or predefined value in the component?